larray/core/array.py

```diff
@@ -7134,25 +7182,31 @@ def zeros(axes, title='', dtype=float, order='C'):
          BE  0.0  0.0
          FO  0.0  0.0
     """
+    if title is not None:
+        import warnings
+        warnings.warn("Argument title is deprecated. Use meta instead", FutureWarning, stacklevel=2)
```

Comment:
when title is provided, and shouldn't we use `meta['title'] = title; title = None` (and initialize meta if None)? Otherwise, you will get twice the same warning: once in zeros (or other array creation functions) then once from `LArray.__init__`. You probably want to make an helper function for this to avoid duplication this code over and over.